bcrs-ui/src/main.ts

```diff
@@ -0,0 +1,75 @@
+import 'core-js/stable' // to polyfill ECMAScript features
+import 'regenerator-runtime/runtime' // to use transpiled generator functions
+
+// Vue Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import router from './router'
+import { store } from '@/store'
+
+// Styles
+import '@/assets/styles/base.scss'
+import '@/assets/styles/layout.scss'
+import '@/assets/styles/overrides.scss'
+import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
+
+// Base App
+import App from './App.vue'
+
+// Helpers
+import { fetchConfig, haveKcTokens } from '@/utils'
```

Comment:
Good decision to import these in this way!

Reply:
Cool, thanks! Glad you like it. Getting more bang for our buck from that utils folder!